Cargo.toml

```diff
@@ -55,8 +55,5 @@ log = "0.4"
 semver = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-serialport = "4.3"
+serialport = "4.7.1"
 thiserror = "1.0"
-
-[patch.crates-io]
-serialport = { git = "https://github.com/haileys/serialport-rs", branch = "tangara-flasher" }
```

Comment:
Was https://github.com/haileys/serialport-rs/commit/109e5e9aaf909bd60a8dc14d641f06911d2754ac ever upstreamed? Or is it not relevant with espflash 3? I don't have Windows easily available to test.